scripts/mutect2_wdl/README.md

```diff
@@ -33,10 +33,10 @@ Recommended default values (where possible) are found in ``mutect2_multi_sample_
 - ``Mutect2_Multi.ref_dict`` -- For Broad internal VM:  ``/seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.dict``
 - ``Mutect2_Multi.pon`` -- (optional) Panel of normals VCF to use for false positive reduction.
 - ``Mutect2_Multi.pon_index`` -- (optional, but required if ``Mutect2_Multi.pon`` is specified)  VCF index for the panel of normals.  Please see GATK4 tool ``IndexFeatureFile`` for creation of an index.
-- ``Mutect2_Multi.dbsnp`` -- (optional)  dbSNP VCF file for pruning of false positives.
-- ``Mutect2_Multi.dbsnp_index`` -- (optional, but required if ``Mutect2_Multi.dbsnp`` is specified)  VCF index for dbSNP.  Please see GATK4 tool ``IndexFeatureFile`` for creation of an index.
-- ``Mutect2_Multi.cosmic`` -- (optional)  COSMIC VCF file to inform potential true positives.
-- ``Mutect2_Multi.cosmic_index`` -- (optional, but required if ``Mutect2_Multi.cosmic`` is specified)  VCF index for COSMIC.  Please see GATK4 tool ``IndexFeatureFile`` for creation of an index.
+- ``Mutect2_Multi.gnomad`` -- (optional)  gnomAD vcf containing population allele frequencies (AF) of common and rare alleles.  Essential for determining possible germline variants in tumor-only calling and helpful in tumor-normal calling as well.
```

Comment:
Can we get suggested download locations?  For the following:
- gnomAD
- variants for contamination

After all, these are optional, but performance really suffers without these (in my experience).